### ISSUE TITLE
Running generate-changelog always drops a procedure before creating it

### ISSUE THREAD

**Issue by tati-qalified** [liquibase/liquibase#5411]
### Search first

- [X] I searched and no similar issues were found

### Description

When running either `generate-changelog` or `diff-changelog` into a sql format, for databases PostgreSQL, Oracle and MySQL, procedures are always dropped and then created:
```
-- changeset tfernandez:1702560678224-5 splitStatements:true
DROP PROCEDURE IF EXISTS "procedure1";
CREATE OR REPLACE PROCEDURE public.procedure1()
 LANGUAGE plpgsql
AS $procedure$
	BEGIN
		insert into public.entities (id, name) values (1, 'two');
	END;
$procedure$;
```

Only for PostgreSQL, views have the same behaviour:
```
DROP VIEW IF EXISTS "newview";
CREATE VIEW "newview" AS SELECT id,  name
   FROM entities;
```

This doesn't happen for any other changelog format.

### Steps To Reproduce

1. Have a PostgreSQL database with a procedure and a view
2. Run `liquibase generate-changelog --changelog-file=generated.sql`
3. Using the same database as reference, run `liquibase diff-changelog --changelog-file=differences.sql`

### Expected/Desired Behavior

The output shouldn't contain "DROP X IF EXISTS"

### Liquibase Version

4.24.0

### Database Vendor & Version

PostgreSQL, Oracle, MySQL 8.0

### Liquibase Integration

All

### Are you willing to submit a PR?

- [ ] I'm willing to submit a PR (Thank you!)

**Comment by tati-qalified** [comment]
@kevin-atx PostgreSQL and Oracle support `create or replace` for both procedures and views.
MySQL doesn't, but it supports `create if (not) exists`.

So the SQL code **could** change the `drop procedure if exists` part. I suppose that what's left is to figure out if we want the behaviour to be **with** `drop` or without, and make all changelog formats behave in the same way.

**Comment by kevin-atx** [comment]
Could you re-test this ticket with the 4.26.0 release? Here are some improvements that might impact this issue:
* https://github.com/liquibase/liquibase/pull/4635
* https://github.com/liquibase/liquibase/pull/3581
* https://github.com/liquibase/liquibase/pull/5304

**Comment by tati-qalified** [comment]
@kevin-atx fixed by 4.26.0
